app/javascript/src/App.jsx

```diff
@@ -1,6 +1,6 @@
 import React from "react";
 
-import dayjs from "src/lib/dayjs"; // eslint-disable-line
+import dayjs from "lib/dayjs"; // eslint-disable-line
```

Comment:
I believe I had already pointed out the issue with this code once before. Check the Slack channel thread.